examples/otel-logs-splunk-raw-hec/README.md

```diff
@@ -0,0 +1,31 @@
+# Splunk Raw HEC example
+
+This example showcases how the collector can expose a HTTP endpoint to receive data over the HEC endpoint as a raw payload.
+
+To learn more about HEC raw capabilities, head on to the [HEC documentation](https://docs.splunk.com/Documentation/Splunk/8.2.6/Data/FormateventsforHTTPEventCollector#Raw_event_parsing).
```

Comment:
Suggested change: "For information about about the raw capabilities of HEC, see Raw event parsing."